data/kahet/beyond patir assets.txt

```diff
@@ -0,0 +1,2460 @@
+
+effect "jump drive 6"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 7"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 8"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 9"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 10"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 11"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 12"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 13"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 14"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 15"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive red"
+	sprite "effect/jump drive red"
+		"no repeat"
+		"frame rate" 8
+		"rewind"
+	"lifetime" 8
+	"velocity scale" 3
+
+effect "jump drive circle"
+	sprite "effect/jd detainer"
+		"no repeat"
+		"frame time" 1.5
+	"lifetime" 50
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 1"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 2"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 3"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 4"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 5"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 6"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 7"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 8"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 9"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+outfit "Asteroid Weapon Large 4"
+	category "Guns"
+	cost 57165239
+	thumbnail "outfit/asteroid weapon 4"
+	"mass" 33
+	"outfit space" -33
+	"weapon capacity" -33
+	"gun ports" -1
+	"asteroid mount 4" -1
+	weapon
+		sprite "effect/asteroid projectile print"
+			"frame rate" 21
+		"velocity" 100
+		"drag" .1
+		"acceleration" 1
+		"lifetime" 20
+		"turn" -6
+		"submunition" "wep1 print" 2
```

Comment:
Do you think these massive submunition chains could be improved if I finished #9832?

Reply:
I think so? Honestly, these weapon codes are so messy and made by so many parts (some of which taken from Fero and Todd) that I can't say for certain without testing

Reply:
I'm fine with leaving these definitions where they're at if they're working already, but I'll revive that PR at some point and we can look at improving things here after it's merged.